src/main/java/dmytro/hadiuchko/springboot/repository/book/BookSpecificationBuilder.java

```diff
@@ -14,6 +14,7 @@ public class BookSpecificationBuilder implements SpecificationBuilder<Book> {
     private static final String AUTHOR_KEY = "author";
     private static final String TITLE_KEY = "title";
     private static final String ISBN_KEY = "isbn";
+    private static final String CATEGORY_KEY = "category";

```

Comment:
No reason to duplicate constants you can reuse them from SpecificationProvider`s